### ISSUE TITLE
rename build name from bacserv to app-amd64 or app-armv7

### ISSUE THREAD

**Issue by NubeDev** [NubeIO/bacnet-server-c#12]
rename the binary but keep zip name the as is

**Comment by shomaglasang** [comment]
@NubeDev hey mate, this is already implemented starting v0.0.4 https://github.com/NubeIO/bacnet-server-c/releases/tag/v0.0.4
```
8637974227 -rwxr-xr-x@ 1 sho  staff  581608 Jul 26 20:13 app-amd64
8637974235 -rwxr-xr-x@ 1 sho  staff  420884 Jul 26 20:25 app-armv7
8637974186 -rw-r--r--@ 1 sho  staff  228149 Jul 26 20:26 bacnet-server-c-0.0.4-a1e608ed.amd64.zip
8637974202 -rw-r--r--@ 1 sho  staff  200391 Jul 26 20:26 bacnet-server-c-0.0.4-a1e608ed.armv7.zip
```

**Comment by shomaglasang** [comment]
Done.
